hack/config

```diff
@@ -10,6 +10,7 @@ HPP_VERSION="v0.7.0"
 CONVERSION_CONTAINER_VERSION="v2.0.0"
 VMWARE_CONTAINER_VERSION="v2.0.0-9"
 VM_IMPORT_VERSION="v0.2.5"
+HCO_KV_IO_VERSION="1.4.0"
```

Comment:
Please remove this one. Use `source ./hack/config`, then you can use the `CSV_VERSION` env. But please do not constant it here.

Reply:
Done.